src/app/components/case/Workflow/Workflow.tsx

```diff
@@ -39,18 +43,20 @@ const Workflow: React.FC<Props> = ({ id }) => {
       <Spinner /> :
     mappedData !== undefined ?
       <>
-      { mappedData.map(([title, information, tasks]) =>
-        <>
-          <Heading as="h4">{ title }</Heading>
-          { information && <Div>{ information }</Div> }
+      { mappedData.map(([title, information, tasks], index) =>
+        <Wrap key={ index }>
```

Comment:
Only using index could go wrong right? Maybe combine it with title?

Reply:
Done